xml/tuning_cgroups.xml

```diff
@@ -57,6 +57,29 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec-tuning-cgroups-accounting">
+  <title>Resource accounting</title>
+  <para>
+  The placement of processes into different cgroups can be used to obtain
+  per-cgroup information of certain resource consumptions.
+  </para>
+  <para>
+  The accounting has relatively small but non-zero overhead whose impact
```

Comment:
This is a bit confusing, is this correct? "Be aware that turning on  accounting for one unit will also implicitly turn it on for all units contained in the same
slice and for all its parent slices and the units contained therein." I copied it from man systemd.resource-control.

Reply:
The quote from the man page is correct (it's not all units in the same slice but just direct descendants of the given slice). That is what I intended to explain in the second sentence of this paragraph. The overhead in the first sentence refers to overhead that exists just because of putting a process in a cgroup (i.e. turning accounting on). Does it clarify what I meant? Should the docs wording be different to express this?

Reply:
Yes please re-word. The accounting overhead sentence is clear, it's the bit about slices and units that confused me. It is fine to copy the man page, or you might want to say it differently. Thanks!

Reply:
I see. I've force-pushed an updated version.